src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -344,6 +381,31 @@ protected final void handleSelectedKey(final SelectionKey sk) throws IOException
 
   }
 
+  private void handleFullTasksCircularBuffer(Runnable r, int sleepMillis)
+  {
+    final Thread currentThread = Thread.currentThread();
```

Comment:
It is not needed. I am not sure why it was introduced, likely `currentThread` was used in other places while I was working on the functionality, but it is not used in the final version.